packages/core/src/shared/utilities/zipStream.ts

```diff
@@ -150,4 +154,13 @@ export class ZipStream {
             streamBuffer: this._streamBuffer,
         }
     }
+
+    public static async unzip(zipBuffer: Buffer): Promise<ZipReaderResult[]> {
```

Comment:
This isn't a streaming interface. So consumers of this function will be limited by available memory.